app/models/user_track.rb

```diff
@@ -78,6 +78,12 @@ def unlocked_exercises_for_exercise(exercise)
   end
 
   def enabled_exercises(exercises)
+    # This should really be "If the user a maintainer FOR THIS TRACK".
+    # I don't know how well we keep that up to date. If we do, then
+    # let's take that approach and refine it a bit. Otherwise let's
+    # consider if we want this brute-force approach for all maintainers.
+    return exercises if user.maintainer?
```

Comment:
There is some code to keep track of that, but we never got around to get it working.
It would not be hard to do so though, so maybe we do that as a follow-up?

Reply:
Yep.